bin/ch/WScriptJsrt.cpp

```diff
@@ -1073,6 +1074,114 @@ JsValueRef __stdcall WScriptJsrt::LoadTextFileCallback(JsValueRef callee, bool i
     return returnValue;
 }
 
+int JsFgets(char* buf, int size, FILE* file)
+{
+    int n = size - 1;
+    if (n < 0)
+        return -1;
+
+    bool crflag = false;
+    int c, i = 0;
+
+    for (i = 0; i < n && (c = getc(file)) != EOF; i++) {
```

Comment:
>{ [](start = 55, length = 1)

nit pick: Could you please change this "{ at end of line" style to "{ at next line by its own", to be consistent with rest of the code?